src/reducers/auth.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+import type {Action} from '../actions/types';
+
+type AuthState = {
+  state: 'mystery' | 'logged in' | 'logged out' | 'login failed',
+  user?: {}, // TODO(marcos): fill this in once you know the shape,
+  errors?: {}
+};
+
+const defaultState = {
+  state: 'mystery',
+};
+
+export default (state: AuthState = defaultState, action: Action) => {
+  // TODO(marcos): fill these in
+  switch(action.type) {
+    case 'AUTH_MYSTERY':
+    case 'ATTEMPT_LOGIN':
+    case 'LOGGED_IN':
+    case 'LOGIN_FAILURE':
+    case 'LOGGED_OUT':
+      return state;
+  }
+  return state;
```

Comment:
with _all_ reducers if the reducer has no idea what type it has just seen, it must always return `state`. if you don't return `state`, not only will `combineReducers` yell at you, you run the risk of nuking your state when handling an unknown action (which is bad)